src/components/sidebar.jsx

```diff
@@ -0,0 +1,301 @@
+import { IoChatbubblesSharp } from "react-icons/io5";
+import { IoMan } from "react-icons/io5";
+import { RiGalleryFill } from "react-icons/ri";
+import { FaPaintBrush } from "react-icons/fa";
+import ReactTooltip from "react-tooltip";
+import { Spin as Hamburger } from "hamburger-react";
+
+const StyledNav = styled.nav`
+    height: 100%;
+    width: 85px;
+    float: left;
+    background-color: #382d3b;
+    box-shadow: 2px 0 7px 0 rgba(0, 0, 0, 0.4);
+    position: fixed;
+    top: 0;
+    overflow-x: hidden;
+    margin: 0 0 0 -110px;
+
+    @media screen and (max-width: 900px), (max-height: 640px) {
+        width: 100%;
+        position: sticky;
+        height: none;
+        margin: 0 0 0 0;
+        display: flex;
+        justify-content: center;
+        transition: display 1s, visibility 1s;
+    }
+`;
+
+const StyledMobileNav = styled.nav`
+    display: none;
+
+    @media screen and (max-width: 900px), (max-height: 640px) {
+        display: ${(props) => props.mobileMenuDisplay};
+        opacity: 1;
+        max-width: 350px;
+        width: 100%;
+        height: auto;
+        top: 80px;
+        position: fixed;
+        margin: auto;
+        background-color: #382d3b;
+        box-shadow: 0 0 7px 0 rgba(0, 0, 0, 0.4);
+        animation: enterMobileMenu 1s;
+
+        @keyframes enterMobileMenu {
+            0% {
+                top: 0;
+                opacity: 0;
+            }
+            100% {
+                top: 80px;
+                opacity: 1;
+            }
+        }
+    }
+`;
+
+const StyledUL = styled.ul`
+    margin: 0 auto;
+    padding: 0;
+    text-align: center;
+
+    @media screen and (max-width: 900px), (max-height: 640px) {
+        width: 100%;
+    }
+`;
+
+const StyledMobileUL = styled.ul`
+    @media screen and (max-width: 900px), (max-height: 640px) {
+        width: 100%;
+        margin: 0 auto;
+        padding: 0;
+        text-align: center;
+    }
+`;
+
+const StyledLI = styled.li`
+    display: inline-block;
+    padding: 5px 0;
+
+    @media screen and (max-width: 900px), (max-height: 640px) {
+        display: none;
+    }
+`;
+
+const StyledMobileLI = styled.li`
+    display: none;
+
+    ${(props) =>
+        props.menuButton &&
+        css`
+            @media screen and (max-width: 900px), (max-height: 640px) {
+                display: inline-block;
+                float: right;
+                padding: 10px 20px;
+            }
+        `}
+
+    ${(props) =>
+        props.logo &&
+        css`
+            @media screen and (max-width: 900px), (max-height: 640px) {
+                display: inline-block;
+                float: left;
+                padding: 10px 20px;
+            }
+        `}
+
+    ${(props) =>
+        props.menu &&
+        css`
+            @media screen and (max-width: 900px), (max-height: 640px) {
+                display: block;
+                padding: 10px 30px;
+            }
+        `}
+`;
+
+const NavLink = styled(BaseNavLink)`

```

Comment:
This is a bit confusing. You imported `NavLink` as `BaseNavLink` just so you can rename it `NavLink` again after styling it? It makes more sense to just import `NavLink`, and then when you add styling to it, call it `StyledNavLink` or `NavLinkStyled`.